draft-ietf-tls-exported-authenticator.md

```diff
@@ -124,17 +124,22 @@ SHOULD use TLS as its underlying transport to keep the request confidential.  Th
 application MAY use the existing TLS connection to transport the authenticator.
 
 An authenticator request message can be constructed by either the client or the
-server.  This authenticator request uses the CertificateRequest message structure
-from Section 4.3.2 of {{!TLS13=RFC8446}}, even if the TLS connection protocol
-is TLS 1.2.  The CertificateRequest is used to define the parameters in a request for an
-authenticator.  This message does not include any TLS framing and is not encrypted
-with a handshake key.
+server.  Server-generated authenticator requests use the CertificateRequest
+message from Section 4.3.2 of {{!TLS13=RFC8446}}. Client-generated
+authenticator requests use a new message, called the ClientCertificateRequest,
+which uses the same structure as CertificateRequest.  These messages
+structures are used even if the TLS connection protocol is TLS 1.2.
 
-The uniqueness requirements of the certificate_request_context apply
-only to CertificateRequest messages that are used as part of authenticator requests.
-There is no impact if the value of a certificate_request_context used in an authenticator
-request matches the value of a certificate_request_context in the handshake or
-in a post-handshake message.  The structure is defined to be:
+The CertificateRequest and ClientCertificateRequest messages are used to define the
+parameters in a request for an authenticator.  These messages do not include any
+TLS framing and are not encrypted with a handshake key.
+
+The structures are defined to be:
+
+       struct {
+          opaque certificate_request_context<0..2^8-1>;
+          Extension extensions<2..2^16-1>;
+       } ClientCertificateRequest;
```

Comment:
If we don't use the handshake-layer framing or allocate a handshake message type for this, is there really any value in giving a new named structure?

Reply:
My impression of the discussion in Singapore was that we would define a new handshake message type for this.  Maybe I'm misremembering though.  It was a long week.

Reply:
I think the outcome was that there'd be no new handshake message type, so I think this is fine as is.